middlewares/replace_path.go

```diff
@@ -15,6 +16,12 @@ const ReplacedPathHeader = "X-Replaced-Path"
 
 func (s *ReplacePath) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	r.Header.Add(ReplacedPathHeader, r.URL.Path)
+	log.Debugf("Replacing Path %s with %s", r.URL.Path, s.Path)
```

Comment:
Could you remove this Debug log? This is way too verbose to keep it in production code :)